packages/next-auth/package.json

```
@@ -84,7 +84,7 @@
     "@simplewebauthn/server": "^9.0.2",
     "next": "^14",
     "nodemailer": "^6.6.5",
-    "react": "^18.2.0"
+    "react": "^18.2.0 || ^19.0.0-0"
```

This should cover more `19.x` things for now and in the future

```suggestion
    "react": "^18.2.0 || ^19"
```

---

AFAIK caret-only `^` doesn't cover the pre-release as well, it had to be set explicitly with `-`.

x-ref: https://semver.npmjs.com

---

Ah okay, makes sense. Thanks for the npm semver calc link

---

Thank you for reviewing!